sequential/doc.go

```diff
@@ -0,0 +1,15 @@
+// Package sequential provides a set of functions for managing sequential
+// files on Windows.
+//
+// The origin of these functions are the golang OS and windows packages,
+// slightly modified to only cope with files, not directories due to the
+// specific use case.
```

Comment:
Not new in this PR, but probably we have to replicate the original copyright header

Reply:
Ah, good point; perhaps extract the bits that were copied to a separate file (I'd have to check how much was copied; I think that's only in the `CreateTemp`, which uses similar /same logic as `os.CreateTemp`)

We can look at that in a follow up in deed 👍

Reply:
FWIW; was also discussing with @crazy-max to see to what extend we still need that function; it may be that some parts were needed for "pre go1.9" code, which didn't work well with Windows, but some of those reasons may no longer be there, so possibly we can start to remove some of this.

I think it's still useful to first do the migration, and go from there though (proposing to tag it as `v0.5`, so that any deprecation that would happen could be done before `v1.0.0`).